service_container
App\Controller\BookController
App\Controller\Controller
App\Controller\SecurityController
App\Other\Constraint\Core\Resolver
Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController
Symfony\Bundle\FrameworkBundle\Controller\RedirectController
Symfony\Bundle\FrameworkBundle\Controller\TemplateController
cache.app_clearer
cache.global_clearer
cache.system_clearer
cache_warmer
console.command_loader
container.env_var_processors_locator
container.get_routing_condition_service
debug.error_handler_configurator
error_controller
lexik_jwt_authentication.encoder
lexik_jwt_authentication.generate_token_command
lexik_jwt_authentication.jwt_manager
lexik_jwt_authentication.key_loader
monolog.logger.deprecation
routing.loader
services_resetter
web_profiler.controller.exception_panel
web_profiler.controller.profiler
web_profiler.controller.router
.container.private.profiler
.virtual_request_stack
cache.app
cache.security_is_granted_attribute_expression_language
cache.system
cache.validator_expression_language
data_collector.cache
data_collector.dump
debug.stopwatch
doctrine
doctrine.dbal.default_connection
doctrine.orm.default_entity_manager
event_dispatcher
http_kernel
messenger.default_bus
request_stack
router
translator
var_dumper.cloner
profiler
App\Kernel
database_connection
doctrine.orm.entity_manager
kernel
Error: Cannot instantiate interface App\Other\Constraint\Core\Resolver in /var/www/html/var/cache/dev/ContainerUqkmxWP/getResolverService.php:23
Stack trace:
#0 /var/www/html/var/cache/dev/ContainerUqkmxWP/App_KernelDevDebugContainer.php(503): ContainerUqkmxWP\getResolverService::do(Object(ContainerUqkmxWP\App_KernelDevDebugContainer), true)
#1 /var/www/html/vendor/symfony/dependency-injection/Container.php(229): ContainerUqkmxWP\App_KernelDevDebugContainer->load('getResolverServ...')
#2 /var/www/html/vendor/symfony/dependency-injection/Container.php(211): Symfony\Component\DependencyInjection\Container::make(Object(ContainerUqkmxWP\App_KernelDevDebugContainer), 'App\\Other\\Const...', 1)
#3 /var/www/html/src/EventListener/RequestListener.php(55): Symfony\Component\DependencyInjection\Container->get('App\\Other\\Const...')
#4 /var/www/html/var/cache/dev/ContainerUqkmxWP/App_KernelDevDebugContainer.php(793): App\EventListener\RequestListener->__construct(Object(App\Kernel))
#5 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(235): ContainerUqkmxWP\App_KernelDevDebugContainer::ContainerUqkmxWP\{closure}()
#6 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(70): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('kernel.request')
#7 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(276): Symfony\Component\EventDispatcher\EventDispatcher->getListeners('kernel.request')
#8 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(133): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->preProcess('kernel.request')
#9 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#10 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#11 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#12 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#13 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#14 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#15 {main}
Error: Cannot instantiate interface App\Other\Constraint\Core\Resolver in /var/www/html/var/cache/dev/ContainerTlzhJTx/App_KernelDevDebugContainer.php:1054
Stack trace:
#0 /var/www/html/var/cache/dev/ContainerTlzhJTx/App_KernelDevDebugContainer.php(804): ContainerTlzhJTx\App_KernelDevDebugContainer::getRequestListenerService(Object(ContainerTlzhJTx\App_KernelDevDebugContainer))
#1 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(235): ContainerTlzhJTx\App_KernelDevDebugContainer::ContainerTlzhJTx\{closure}()
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(70): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('kernel.request')
#3 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(276): Symfony\Component\EventDispatcher\EventDispatcher->getListeners('kernel.request')
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(133): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->preProcess('kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
Error: Cannot instantiate interface App\Other\Constraint\Core\Resolver in /var/www/html/var/cache/dev/ContainerS1OB3t9/App_KernelDevDebugContainer.php:1114
Stack trace:
#0 /var/www/html/var/cache/dev/ContainerS1OB3t9/App_KernelDevDebugContainer.php(817): ContainerS1OB3t9\App_KernelDevDebugContainer::getRequestListenerService(Object(ContainerS1OB3t9\App_KernelDevDebugContainer))
#1 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(235): ContainerS1OB3t9\App_KernelDevDebugContainer::ContainerS1OB3t9\{closure}()
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(70): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('kernel.request')
#3 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(276): Symfony\Component\EventDispatcher\EventDispatcher->getListeners('kernel.request')
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(133): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->preProcess('kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
ReflectionException: Class "App\Controller\SecurityController::login" does not exist in /var/www/html/src/EventListener/RequestListener.php:85
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(85): ReflectionClass->__construct('App\\Controller\\...')
#1 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
ErrorException: Warning: Undefined variable $container in /var/www/html/src/EventListener/RequestListener.php:49
Stack trace:
#0 /var/www/html/var/cache/dev/ContainerBmWkeFT/App_KernelDevDebugContainer.php(1050): App\EventListener\RequestListener->__construct(Object(App\Kernel), Object(App\Other\Constraint\ConstraintResolver))
#1 /var/www/html/var/cache/dev/ContainerBmWkeFT/App_KernelDevDebugContainer.php(794): ContainerBmWkeFT\App_KernelDevDebugContainer::getRequestListenerService(Object(ContainerBmWkeFT\App_KernelDevDebugContainer))
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(235): ContainerBmWkeFT\App_KernelDevDebugContainer::ContainerBmWkeFT\{closure}()
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(70): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('kernel.request')
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(276): Symfony\Component\EventDispatcher\EventDispatcher->getListeners('kernel.request')
#5 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(133): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->preProcess('kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#7 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#8 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#9 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#10 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#11 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#12 {main}
ReflectionException: Class "App\Controller\SecurityController::login" does not exist in /var/www/html/src/EventListener/RequestListener.php:85
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(85): ReflectionClass->__construct('App\\Controller\\...')
#1 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
ReflectionException: Class "App\Controller\SecurityController::login" does not exist in /var/www/html/src/EventListener/RequestListener.php:76
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(76): ReflectionClass->__construct('App\\Controller\\...')
#1 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
ReflectionException: Class "login" does not exist in /var/www/html/src/EventListener/RequestListener.php:76
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(76): ReflectionClass->__construct('login')
#1 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#2 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#4 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
Symfony\Component\DependencyInjection\Exception\RuntimeException: Cannot resolve argument $request of "App\Controller\SecurityController::login()": Cannot autowire service "App\Request\Security\LoginRequest": argument "$container" of method "App\Request\Core\Request::__construct()" references class "Symfony\Component\DependencyInjection\Container" but no such service exists. in /var/www/html/vendor/symfony/dependency-injection/Container.php:397
Stack trace:
#0 /var/www/html/vendor/symfony/dependency-injection/Argument/ServiceLocator.php(40): Symfony\Component\DependencyInjection\Container->getService('privates', 'App\\Request\\Sec...', NULL, 'Cannot autowire...')
#1 /var/www/html/vendor/symfony/http-kernel/Controller/ArgumentResolver/ServiceValueResolver.php(84): Symfony\Component\DependencyInjection\Argument\ServiceLocator->get('request')
#2 /var/www/html/vendor/symfony/http-kernel/Controller/ArgumentResolver/TraceableValueResolver.php(60): Symfony\Component\HttpKernel\Controller\ArgumentResolver\ServiceValueResolver->resolve(Object(Symfony\Component\HttpFoundation\Request), Object(Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata))
#3 /var/www/html/vendor/symfony/http-kernel/Controller/ArgumentResolver.php(91): Symfony\Component\HttpKernel\Controller\ArgumentResolver\TraceableValueResolver->resolve(Object(Symfony\Component\HttpFoundation\Request), Object(Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata))
#4 /var/www/html/vendor/symfony/http-kernel/Controller/TraceableArgumentResolver.php(40): Symfony\Component\HttpKernel\Controller\ArgumentResolver->getArguments(Object(Symfony\Component\HttpFoundation\Request), Array, Object(ReflectionMethod))
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(173): Symfony\Component\HttpKernel\Controller\TraceableArgumentResolver->getArguments(Object(Symfony\Component\HttpFoundation\Request), Array, Object(ReflectionMethod))
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#7 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#8 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#9 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#10 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#11 {main}
TypeError: App\Request\Core\Request::__construct(): Argument #1 ($kernel) must be of type App\Kernel, ContainerReCjHDK\App_KernelDevDebugContainer given, called in /var/www/html/src/EventListener/RequestListener.php on line 61 and defined in /var/www/html/src/Request/Core/Request.php:54
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(61): App\Request\Core\Request->__construct(Object(ContainerReCjHDK\App_KernelDevDebugContainer), Object(App\Other\Constraint\ConstraintResolver))
#1 /var/www/html/src/EventListener/RequestListener.php(85): App\EventListener\RequestListener->buildRequest('App\\Request\\Sec...')
#2 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#3 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#4 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#5 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#6 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#7 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#8 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#9 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#10 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#11 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#12 {main}
TypeError: App\Controller\SecurityController::__construct(): Argument #2 ($userRepository) must be of type App\Repository\UserRepository, Symfony\Bundle\FrameworkBundle\Routing\Router given, called in /var/www/html/src/EventListener/RequestListener.php on line 90 and defined in /var/www/html/src/Controller/SecurityController.php:26
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(90): App\Controller\SecurityController->__construct(Object(Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager), Object(Symfony\Bundle\FrameworkBundle\Routing\Router), Object(Symfony\Component\HttpFoundation\RequestStack), Object(ContainerQshxObf\EntityManagerGhostEbeb667), Object(App\Other\Constraint\ConstraintResolver), 'App\\Controller\\...', 'login')
#1 /var/www/html/src/EventListener/RequestListener.php(77): App\EventListener\RequestListener->buildRequest()
#2 /var/www/html/src/EventListener/RequestListener.php(104): App\EventListener\RequestListener->wire()
#3 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#4 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#5 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#6 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#7 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#8 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#9 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#10 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#11 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#12 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#13 {main}
TypeError: App\Request\Core\Request::__construct(): Argument #1 ($requestStack) must be of type Symfony\Component\HttpFoundation\RequestStack, Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager given, called in /var/www/html/src/EventListener/RequestListener.php on line 91 and defined in /var/www/html/src/Request/Core/Request.php:52
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(91): App\Request\Core\Request->__construct(Object(Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager), Object(Symfony\Bundle\FrameworkBundle\Routing\Router), Object(Symfony\Component\HttpFoundation\RequestStack), Object(ContainerQshxObf\EntityManagerGhostEbeb667), Object(App\Other\Constraint\ConstraintResolver), 'App\\Controller\\...', 'login', 'App\\Request\\Sec...')
#1 /var/www/html/src/EventListener/RequestListener.php(78): App\EventListener\RequestListener->buildRequest()
#2 /var/www/html/src/EventListener/RequestListener.php(105): App\EventListener\RequestListener->wire()
#3 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#4 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#5 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#6 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#7 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#8 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#9 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#10 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#11 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#12 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#13 {main}
TypeError: App\Request\Core\Request::__construct(): Argument #1 ($requestStack) must be of type Symfony\Component\HttpFoundation\RequestStack, Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager given, called in /var/www/html/src/EventListener/RequestListener.php on line 91 and defined in /var/www/html/src/Request/Core/Request.php:52
Stack trace:
#0 /var/www/html/src/EventListener/RequestListener.php(91): App\Request\Core\Request->__construct(Object(Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager), Object(Symfony\Bundle\FrameworkBundle\Routing\Router), Object(Symfony\Component\HttpFoundation\RequestStack), Object(ContainerL0oC9s0\EntityManagerGhostEbeb667), Object(App\Other\Constraint\ConstraintResolver), 'App\\Controller\\...', 'login', 'App\\Request\\Sec...')
#1 /var/www/html/src/EventListener/RequestListener.php(78): App\EventListener\RequestListener->buildRequest()
#2 /var/www/html/src/EventListener/RequestListener.php(105): App\EventListener\RequestListener->wire()
#3 /var/www/html/vendor/symfony/event-dispatcher/Debug/WrappedListener.php(116): App\EventListener\RequestListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#4 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(220): Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request', Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher))
#5 /var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php(56): Symfony\Component\EventDispatcher\EventDispatcher->callListeners(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\RequestEvent))
#6 /var/www/html/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php(139): Symfony\Component\EventDispatcher\EventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#7 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(157): Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch(Object(Symfony\Component\HttpKernel\Event\RequestEvent), 'kernel.request')
#8 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#9 /var/www/html/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#10 /var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#11 /var/www/html/vendor/autoload_runtime.php(29): Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
#12 /var/www/html/public/index.php(5): require_once('/var/www/html/v...')
#13 {main}
